libkbfs/state_checker.go

```diff
@@ -93,13 +93,15 @@ func (sc *StateChecker) getLastGCData(ctx context.Context,
 
 	var latestTime time.Time
 	var latestRev kbfsmd.Revision
-	for _, c := range *config.allKnownConfigsForTesting {
-		ops := c.KBFSOps().(*KBFSOpsStandard).getOps(context.Background(),
-			FolderBranch{tlfID, MasterBranch}, FavoritesOpNoChange)
-		rt, rev := ops.fbm.getLastQRData()
-		if rt.After(latestTime) && rev > latestRev {
-			latestTime = rt
-			latestRev = rev
+	if config.allKnownConfigsForTesting != nil {
```

Comment:
I _think_ this is the right answer (because kbfsdocker runs in devel mode), but I'm not sure. @strib?

Reply:
This actually isn't necessary, I think. I noticed a panic here at first, so I added this check like elsewhere. But it turned out it was because we were using a wrong MDServer; now that it's fixed this should never happen. But I'm fine with either having this or not, so let me know if any of you want to leave it out so that we have an intentional panic if this-should-never-happen happens.